pkg/ocm/share/repository/nextcloud/nextcloud.go

```diff
@@ -182,23 +186,27 @@ func (sm *Manager) GetShare(ctx context.Context, user *userpb.User, ref *ocm.Sha
 	log.Debug().Msgf("Found ShareAltMap %s", rejson)
 	return &ocm.Share{
 		Id: altResult.ID,
-		ResourceId: &provider.ResourceId{
-			OpaqueId:  altResult.ResourceID.OpaqueID,
-			StorageId: sm.mountID,
-		},
+		// ResourceId: &provider.ResourceId{
+		// 	OpaqueId:  altResult.ResourceID.OpaqueID,
+		// 	StorageId: sm.mountID,
+		// },
+		ResourceId: nil,
```

Comment:
This was added to fix the access to remote resources, if you remove it the access is broken. Please revert ;)